<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGameOver</class>
 <widget class="QDialog" name="CGameOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="headline">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Game Over</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rankLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="nameLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="roundLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="countryLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="unitsLayout"/>
  </widget>
  <widget class="QLabel" name="rankLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Rang</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="roundLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Runde</string>
   </property>
  </widget>
  <widget class="QLabel" name="countryLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Länder</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitsLabel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Einheiten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitPB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
